modules/web/src/app/node-data/basic/provider/nutanix/template.html

```diff
@@ -22,7 +22,8 @@
            matInput
            type="text"
            autocomplete="off"
-           required>
+           required
+           valueChanged>
```

Comment:
We should also handle value changed for Categories in this component which is similar to label-form and taint-form component.

Reply:
Its also missing for `CPU Passthrough` checkbox.

Reply:
right now nutanix is not available so we can't implement it without test it